samples/highcharts/demo/column-test/demo.details

```diff
@@ -0,0 +1,14 @@
+---
+name: Basic column
```

Comment:
Should probably give the files & demo a suitable name. We already have basic column, but maybe "Single columns" makes sense?